src/browser.ts

```diff
@@ -67,6 +70,7 @@ export const launchBrowser = async (browserWindow: BrowserWindow) => {
       `--proxy-server=http://localhost:${appSettings.proxy.port}`,
       `--ignore-certificate-errors-spki-list=${certificateSPKI}`,
       disableChromeOptimizations,
+      url ?? '',
```

Comment:
This looks sketchy tbh, but I couldn't find a better alternative. Do you think it should somehow be sanitized before getting to this line?
The decision not to include any validation was deliberate in that it's much easier to correct erroneous input in the browser and continue the recording than to read a validation message and re-submit the form.

Reply:
I tried to input a command like `&& echo "HELLO"` but it stills opens it in chrome as url, personally I think it looks good 🙌

Reply:
This seems to be sanitized behind the scenes by Chrome. I tested a few cases including `javascript:alert('')`.

We may want to sanitize this URL if we end up storing it in settings or something, but it looks good!